chevah_build

```diff
@@ -264,7 +263,8 @@ case $OS in
         # and the result is not portable.
         export BUILD_LIBEDIT="no"
         ;;
-    sles10)
+    sles10|sles11|rhel5)
+        # Up-to-date cryptography not working on these...
```

Comment:
I don't know what to say about sles 11 and rhel5.

this change will brink back ancient pyopenssl version.

I think that is better to add a separate condition for them and set PIP_LIBRARIES to the  pyOpenSSL==16.0.0 and cryptography==1.3.1